### PR TITLE
MH-131: refactoring

### PR DESCRIPTION
**Pull request by iacobfred**

### CODE REVIEW

File: apps/moderation/serializers.py [hunk 1]
```diff
@@ -51,8 +51,8 @@ def get_moderated_fields(self) -> list[dict]:
                     'name': field_name,
                     'type': field.__class__.__name__,
                     'editable': not getattr(field, 'read_only', False),
-                    'required': not getattr(field, 'required', False),
```
Comment (alashow) [inline apps/moderation/serializers.py:54]: Where did these two `not`s come from? 🤔

Comment (iacobfred) [reply apps/moderation/serializers.py:54]: No idea. Noticed them while discussing fields API behavior with @mattfredericksen.

File: core/settings.py [hunk 2]
```diff
@@ -53,7 +53,7 @@
 # https://docs.djangoproject.com/en/dev/ref/settings#s-allowed-hosts
 ALLOWED_HOSTS = config(
     'ALLOWED_HOSTS',
-    default='localhost, 127.0.0.1, 0.0.0.0, django',
+    default='localhost, 127.0.0.1, django',
```
Comment (alashow) [inline core/settings.py:56]: I hope this is intentional?

**Review by alashow — APPROVED** [review]
LGTM

**Pull request merged**